### PR TITLE
More robust layout management

### PR DESCRIPTION
**Pull request by ancorgs**

### CODE REVIEW

**Comment by imobach** [comment]
LGTM

File: src/app/components/notifier/notifier.service.js [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
     function info(text) {
       notifications.info("Information", text, {
-        duration: 20000,
+        duration: 5000,
```
Comment (jreidinger) [inline src/app/components/notifier/notifier.service.js]: Reason why I use so big number is to allow user to move mouse and click button, for older users like me it is hard to notice notification, catch mouse and then move it there :)

Comment (ancorgs) [inline src/app/components/notifier/notifier.service.js]: Yes. That's ok for `MuteNotifier`. This one is regular `Notifier`, where user interaction is not expected.

Comment (jreidinger) [inline src/app/components/notifier/notifier.service.js]: ah, ok, I miss it, that it is not Mute one.

**Pull request merged**